lib/utils.js

```diff
@@ -540,8 +540,8 @@ function jsonStringify (object, spaces, depth) {
     --length;
     str += '\n ' + repeat(' ', space) +
       (isArray(object) ? '' : '"' + i + '": ') + // key
-      _stringify(object[i]) +                    // value
-      (length ? ',' : '');                       // comma
+      _stringify(object[i]) + // value
+      (length ? ',' : ''); // comma
```

Comment:
[unimportant] I long for the day that linters can recognise when whitespace has been properly used to make things, any things, line up across multiple lines. [/unimportant]